src/network/ConnectionManager.ts

```diff
@@ -148,24 +161,15 @@ export class ConnectionRegistryImpl implements ConnectionRegistry {
 
     private active = false;
     private readonly activeConnections = new Map<string, Connection>();
-    private readonly loadBalancer: LoadBalancer;
     private connectionState = ConnectionState.INITIAL;
-    private readonly smartRoutingEnabled: boolean;
-    private readonly asyncStart: boolean;
-    private readonly reconnectMode: ReconnectMode;
-    private readonly clusterService: ClusterService;
 
     constructor(
-        connectionStrategy: ConnectionStrategyConfig,
-        smartRoutingEnabled: boolean,
-        loadBalancer: LoadBalancer,
-        clusterService: ClusterService
+        private readonly asyncStart: boolean,
+        private readonly reconnectMode: ReconnectMode,
+        private readonly smartRoutingEnabled: boolean,
+        private readonly loadBalancer: LoadBalancer,
+        private readonly clusterService: ClusterService
```

Comment:
preferred to use compact construction syntax(like other classes I refactored in #797 )